api_tests/harness.js

```diff
@@ -228,9 +228,7 @@ describe("🛠", async function() {
         await generateBlock(config.ledgers, ledgers_config);
     });
 
-    describe("👟", async function() {
-        require(test_dir + "/test.js");
-    });
+    require(test_dir + "/test.js");
```

Comment:
❓ are you sure you are having reliable test run with this change?

Reply:
Haven't has any problems so far.